packages/core/src/modules/indy/repository/AnonCredsCredentialDefinitionRecord.ts

```diff
@@ -1,39 +1,54 @@
+import type { DidIndyNamespace } from '../../../utils/indyIdentifiers'
 import type { CredDef } from 'indy-sdk'
 
 import { BaseRecord } from '../../../storage/BaseRecord'
-import { didFromCredentialDefinitionId } from '../../../utils/did'
+import {
+  createQualifiedIdentifier,
+  getDidFromSchemaOrCredentialDefinitionId,
+  createUnqualifiedIdentifier,
+} from '../../../utils/indyIdentifiers'
 
 export interface AnonCredsCredentialDefinitionRecordProps {
   credentialDefinition: CredDef
+  didIndyNamespace: DidIndyNamespace
 }
 
 export type DefaultAnonCredsCredentialDefinitionTags = {
-  credentialDefinitionId: string
-  issuerDid: string
-  schemaId: string
-  tag: string
+  qualifiedIdentifier: string
+  unqualifiedIdentifier: string
 }
 
 export class AnonCredsCredentialDefinitionRecord extends BaseRecord<DefaultAnonCredsCredentialDefinitionTags> {
```

Comment:
I am not sure how I feel about using the `AnonCredsCredentailDefinitionRecord.id` for the qualifiedIdentifier. Every other record, that I am aware of, uses it as an internal tracker and has no actual relation to a "real value".

In the record itself it is stored as `identifier` which could be changed to `resourceIdentifier` to specify that that is the identifier for whatever resource it is stored upon. Having it like this does not make it indy specific right?

Reply:
Any more thoughts on this?

Reply:
I mean modifying the actual id of the schema / cred def.

The schem object has the following structure:

```ts
export interface Schema {
    id: SchemaId;
    attrNames: string[];
    name: string;
    version: string;
    ver: string;
    seqNo: number;
}
```

I'm proposing we modify the schema.id property itself, not the record property. Because this is what the schema.id property will look like once we have did:indy identifiers

Reply:
So not adding any new properties besides the schema / cred def object itself

Reply:
Yeah thats okay by me.